rskj/0.6.2/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:16.04
+
+RUN apt-get update -y && \
+    apt-get install -y git curl gnupg-curl openjdk-8-jdk && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get autoremove -y && \
+    apt-get clean
+
+RUN gpg --keyserver https://secchannel.rsk.co/release.asc --recv-keys 5DECF4415E3B8FA4
+RUN gpg --finger 5DECF4415E3B8FA4
+
+RUN git clone --branch ORCHID-0.6.2 https://github.com/rsksmart/rskj.git /code/rskj
```

Comment:
You can try
`git clone --single-branch --depth 1 --branch ORCHID-0.6.2 git@github.com:rsksmart/rskj.git`